docs/architecture/distributed-cloud-native-apps-containers/introduction-dot-net-aspire/components.md

```diff
@@ -0,0 +1,54 @@
+---
+title: .NET Aspire Components 
+description: Architecture for Distributed Cloud-Native Apps with .NET Aspire & Containers | .NET Aspire Components 
+author: 
+ms.date: 04/25/2024
+---
+
+# .NET Aspire Components: Empowering Cloud-Native Applications
+
+[!INCLUDE [download-alert](../includes/download-alert.md)]
+
+![A diagram showing some example .NET Aspire components - and the NuGet package website.](media/aspire-components.png)
+
+**Figure 3-5**. Examples of .NET Aspire components.
+
+.NET Aspire components are curated **NuGet packages** that handle specific cloud-native concerns. These components connect seamlessly with your app, ensuring consistent integration with services such as Redis, PostgreSQL, and more.
+
+Here is a list of popular .NET Aspire components:
```

Comment:
I guess maybe I would do 2 sections... aspire components and then another list of components specific to azure. This ensures the reader knows you have the ability to use non-azure in aspire projects.